app/app.py

```diff
@@ -129,14 +129,14 @@ def do_work(self, channel, delivery_tag, properties, body):
                 "md5_hash_essence_sidecar": sidecar.md5,
             }
 
-            if md5_hash_essence_manifest != sidecar.md5:
+            if md5_hash_essence_manifest != sidecar.md5.lower():
                 self.log.error(
-                    f"Supplied MD5 differs from the calculated MD5.",
-                    sidecar_md5=sidecar.md5,
+                    "Supplied MD5 differs from the calculated MD5.",
+                    sidecar_md5=sidecar.md5.lower(),
```

Comment:
To `lower` or not to `lower`?

Ik ben voorstander om de gelogde sidecar md5 ook te lowercasen aangezien we de lowercase waarde gebruiken in de check. Dit maakt vergelijken van de twee md5s o.b.v. de logs eenvoudiger en minder verwarrend. 

Maar er valt wel iets te zeggen over het loggen van de "echte" md5 representatie.

Wat is jullie voorkeur?

Reply:
:+1: to lower.